### DISCUSSION TITLE
How to use visual Switch (flow) activities in Elsa 3.1.3 designer?

### DISCUSSION THREAD

**Discussion by shxylgithub** [elsa-workflows/elsa-core#5694] [General]
How to use visual Switch (flow) activities in Elsa 3.1.3 designer?

**Comment by glime-ai** [comment]
To use visual Switch (flow) activities in Elsa 3.1.3 designer, you would typically follow these steps:

1. Open the Elsa Designer interface in your web browser.
2. Create a new workflow or edit an existing one.
3. Drag the "Flow Switch" activity from the activities toolbox into your workflow design area.
4. Configure the "Flow Switch" activity by setting up the cases you want to handle. Each case represents a possible path the workflow can take.
5. Connect other activities to the outcomes of the "Flow Switch" activity to define what should happen for each case.

However, it's important to note that there might be some limitations or issues with the visual designer in Elsa 3.1.3, as indicated by the known issues and ongoing work mentioned in the documentation. If you encounter any problems or the "Flow Switch" activity does not work as expected, you should check the official Elsa documentation, GitHub issues, or community forums for any updates or workarounds.

If you need further assistance or more detailed instructions, please let me know, and I can provide additional guidance.

This comment was generated by [Glime](https://glimelab.ai/?utm_source=dRbvTb7IhCSYiI&utm_medium=github).
